aeron-client/src/main/c/concurrent/aeron_thread.h

```diff
@@ -49,6 +49,12 @@ typedef pthread_t aeron_thread_t;
 #define aeron_thread_get_specific pthread_getspecific
 #define aeron_thread_set_specific pthread_setspecific
 
+typedef pthread_cond_t aeron_cond_t;
```

Comment:
We need something similar to this for Windows as the missing types are causing a compilation failure on Windows.

Reply:
@mikeb01 C11 might a possible option (https://devblogs.microsoft.com/cppblog/c11-threads-in-visual-studio-2022-version-17-8-preview-2/) or https://learn.microsoft.com/en-us/windows/win32/Sync/condition-variables.

Reply:
Windows `CRITICAL_SECTION` and `CONDITION_VARIABLE` will workhere.